src/Transition.tsx

```diff
@@ -95,33 +96,26 @@ const CSSTransition: React.FC<TransitionProps> = function CSSTransition({
 }
 
 const Transition: React.FC<TransitionProps> = function Transition({ show, appear, ...rest }) {
-  const { parent } = useContext(TransitionContext)
+  const { parent } = useContext(transitionContext)
   const isInitialRender = useIsInitialRender()
   const isChild = show === undefined
 
   if (isChild) {
+    return <CSSTransition appear={parent.appear} show={parent.show} {...rest} />
```

Comment:
I wasn't sure what was going on here in Adam's original [gist](https://gist.github.com/adamwathan/e0a791aa0419098a7ece70028b2e641e#file-transition-js-L84). 

I don't totally understand what the `appear` prop is for in `CSSTransition`, but from what I can tell it's a class name, like `enter` and `exit`. It doesn't make sense to `OR` it with a boolean. 

http://reactcommunity.org/react-transition-group/css-transition

I assumed that Adam just goofed there, thinking that `appear` is a boolean - it would be an easy mistake to make. 

As a side note - I wonder if it wouldn't make more sense to just use the Transition component provided by Tailwind's new [headlessui](https://github.com/tailwindlabs/headlessui/tree/develop/packages/%40headlessui-react) project.